srlife/damage.py

```diff
@@ -535,15 +791,67 @@ def calculate_eq_stress(self, mandel_stress, temperatures, material):
         """
 
         # Material parameters
-        nu = material.nu(temperatures).flat[0]
+        nu = np.mean(material.nu(temperatures), axis=0)
 
+        # Normal stress
         sigma_n = self.calculate_normal_stress(mandel_stress)
+
+        # Shear stress
         tau = self.calculate_shear_stress(mandel_stress)
+
         # Projected equivalent stress
         return 0.5 * (
-            sigma_n + np.sqrt((sigma_n**2) + ((tau / (1 - (0.5 * nu))) ** 2))
+            sigma_n
+            + np.sqrt((sigma_n**2) + ((tau / (1 - (0.5 * nu[..., None, None]))) ** 2))
         )
 
+    def calculate_kbar(self, temperatures, material, A, dalpha, dbeta):
```

Comment:
This needs a properly-formatted header like:

```
"""
Calculate the evaluating normalized crack density coefficient (kbar)
using the Weibull scale parameter (svals), Weibull modulus (mvals),
poisson ratio (nu)

Parameters:
    temperatures: current temperatures
etc...
```

Reply:
Written a more descriptive one now.